ui/src/components/Configuration/ReqDefinitionForm.vue

```diff
@@ -1,13 +1,230 @@
 <template>
     <p class="title">New Requisition Definition</p>
+    <div class="p-col-9">
+        <div class="form">
```

Comment:
Instead of this being a class, can it be an actual <form> tag instead of a div? That way it's a valid form from an accessibility standpoint.

Reply:
You will see the change in next PR (PE-58 )